### ISSUE TITLE
Should we support batch norm with gradient accumulation

### ISSUE THREAD

**Issue by AkashGanesan** [ryanccarelli/ssljax#74]
How do we accumulate batch stats across multiple gradient steps? Is this something that is done or does the complexity outweigh the benefits for doing this?
